editor/tests/org.fusesource.ide.camel.editor.tests.integration/META-INF/MANIFEST.MF

```diff
@@ -28,6 +28,8 @@ Require-Bundle: org.junit;bundle-version="4.12.0",
  org.fusesource.ide.camel.model.service.core.tests.integration,
  org.fusesource.ide.branding;bundle-version="9.1.0",
  org.fusesource.ide.projecttemplates;bundle-version="9.1.0",
- org.eclipse.draw2d
+ org.eclipse.draw2d,
+ org.fusesource.ide.camel.tests.util
+Export-Package: org.fusesource.ide.camel.editor.integration
```

Comment:
Right, this was need only when AbstractCamelEditorIT is in editor.test.integration plugin. I will remove this Export.